### PR TITLE
adds a buncha crap to loadout

### PR DESCRIPTION
**Pull request by Belsima**
Standardizes some grammar.

Adds: stethoscopes, lockets, medals, goggles, a white cane, a pair of bags, the powdered wig, a colorable cap, rabbit ears (to go with cat ears. im disgusted.), cheap lighters, greatcoats, and cameras.

Reorganizes some loadout items, and made others selections.

Made the wedding dress the actual wedding dress.

i'm sorry for the commits, at&t went to the wrong house we still have no internet

### CODE REVIEW

**Comment by Belsima** [comment]
fuck it im not fixing travis errors from my phone

**Comment by Belsima** [comment]
wait a sec no i can fix that

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm [hunk 1]
```diff
@@ -209,3 +209,17 @@
...
+
+/datum/gear/accessory/stethoscope
+	display_name = "stethoscope"
+	path = /obj/item/clothing/accessory/stethoscope
+
+/datum/gear/accessory/locket
+	display_name = "locket"
+	path = /obj/item/clothing/accessory/locket
+
+/datum/gear/accessory/medal
```
Comment (Anewbe) [inline code/modules/client/preference_setup/loadout/loadout_accessories.dm]: Don't really want this.

File: code/modules/client/preference_setup/loadout/loadout_general.dm [hunk 2]
```diff
@@ -2,6 +2,19 @@
 	display_name = "cane"
 	path = /obj/item/weapon/cane
 
+/datum/gear/cane/white
+	display_name = "white cane"
+	path = /obj/item/weapon/cane/whitecane
+
+/datum/gear/bag
```
Comment (Anewbe) [inline code/modules/client/preference_setup/loadout/loadout_general.dm]: Don't really want this.

File: code/modules/client/preference_setup/loadout/loadout_general.dm [hunk 3]
```diff
@@ -2,6 +2,19 @@
...
+	display_name = "plastic bag"
+	path = /obj/item/weapon/storage/bag/plasticbag
+	cost = 2
+
+/datum/gear/bag/cash
```
Comment (Anewbe) [inline code/modules/client/preference_setup/loadout/loadout_general.dm]: Don't really want this.

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm [hunk 4]
```diff
@@ -209,3 +209,11 @@
...
 		sweaters[initial(sweater_type.name)] = sweater_type
 	gear_tweaks += new/datum/gear_tweak/path(sortAssoc(sweaters))
+
+/datum/gear/accessory/stethoscope
+	display_name = "stethoscope"
+	path = /obj/item/clothing/accessory/stethoscope
+
+/datum/gear/accessory/locket
+	display_name = "locket"
+	path = /obj/item/clothing/accessory/locke
```
Comment (Anewbe) [inline code/modules/client/preference_setup/loadout/loadout_accessories.dm]: locke

**Pull request closed without merging**